tests/resources/test_gemd_resource.py

```diff
@@ -0,0 +1,160 @@
+import random
+from uuid import UUID
+
+import pytest
+
+from citrine.resources.gemd_resource import GEMDResourceCollection
+from citrine.resources.material_run import MaterialRun
+from citrine.resources.material_spec import MaterialSpec
+
+from tests.utils.factories import MaterialRunFactory, MaterialSpecFactory, MaterialTemplateFactory
+from tests.utils.session import FakeSession, FakeCall
+
+
+@pytest.fixture
+def session() -> FakeSession:
+    return FakeSession()
+
+
+@pytest.fixture
+def collection(session) -> GEMDResourceCollection:
+    return GEMDResourceCollection(
+        project_id=UUID('6b608f78-e341-422c-8076-35adc8828545'),
+        dataset_id=UUID('8da51e93-8b55-4dd3-8489-af8f65d4ad9a'),
```

Comment:
Rather than hard-coding, I suggest using random uuids (`uuid.uuid4()`) and then referring to them by variable if you need to do a comparison (though it doesn't look like any of the tests here check for the specific uuids)

Reply:
Changed in most recent push, along with a few doc updates.